### DISCUSSION TITLE
Missing RCareOMPL class

### DISCUSSION THREAD

**Discussion by jvm1304** [empriselab/RCareWorld#77] [Help - Using] [answered]
I'm trying to work on motion planning using RCareWorld and was interested in using OMPL. According to the documentation in notion, there is an RCareOMPL class and  omplmanager_attr available for use, however I'm having trouble finding it in the github repo. Is this module available for use?

**Comment by YoruCathy** [comment]
Hi @jvm1304 , are you using it for the PhyRC challenge or general usage?

**Reply by jvm1304** [reply]
general usage

**Reply by YoruCathy** [reply, marked as answer]
We are working on updating this module. It will be back soon.
